### ISSUE TITLE
no images found, try another album

### ISSUE THREAD

**Issue by techadvisor1997** [chtaylo2/Roku-GooglePhotos#422]
photoview gets this error often "no images found, try another album".  there is obviosly images in the album and the only way to seem to fix it temporarily is to go into screen saver settings and make sure the album is selected.  This seems to happen on hisense roku tv when the tv is put into hibernation mode but also seems to happen after a number of hours running.

**Comment by techadvisor1997** [comment]
the album has exactly 20 images and i've told photoview app to view them 720p quality.

**Comment by wred2** [comment]
I had the same problem for a few weeks after, I believe, an update to the Roku Ultra a few months ago. Then the symptom changed. Now, the first time the screen saver starts, after sitting for the defined idle period, it displays a blank screen with only the PhotoView logo in the lower right corner. After I click the home button on the Roku remote, it will usually load and display the photos. I say usually, because sometimes I have to go back to the Roku home page two or three times before the photos will load and display.

**Comment by techadvisor1997** [comment]
thank you, i appreciate the feedback. i tracked it down to appear to be a weak wifi signal or slow internet connection issue. the problem occurs when connected to a public wifi accesspoint but connected to a different ssid with a better wifi connection and faster internet, the problem didn't occur. I switched it back to the original wifi ap and problem came back.  the public internet connection has weak signal due the placement of the ap so i'm thinking that is likely the issue.

**Comment by wred2** [comment]
I'm on a gigabit LAN wired connection with a 100Mb connection to the internet.

**Comment by SoundSailor** [comment]
Also on a wired gig connection, also getting no images found messages

**Comment by fdbryant3** [comment]
I have been having the same issue.  Nothing has changed about my network but this problem started a few months ago.  Sometimes it shows the pictures other it says no images found.  I don't suppose there has been a resolution.

**Comment by SoundSailor** [comment]
No resolution that I'm aware of. The slides play perfectly when using the application, just not in screensaver mode. Same behavior on Roku Ultimate (wired GB connection) and Roku Premiere (wireless).

**Comment by chtaylo2** [comment]
The no albums found used to happen often when the daily limit of photos were reached. @SoundSailor / @fdbryant3 - are you seeing the issue as soon as you’re screensaver starts?  I’ve not been able to reproduce this issue. Please help with any other details you can provide.  

Thx

**Comment by SoundSailor** [comment]
Yes, as soon as the screensaver starts. I'm happy to supply any other details that might help you diagnose this.

The app works fine displaying any of my albums when run manually. When set up as a screensaver, it fails immediately on screensaver running. This wasn't always the case, on the Roku Ultra it worked fine for quite a while. It may have been the Roku update that caused the problem, I didn't pay enough attention at the time to know for sure. I got a Roku Premiere when Wally's had a sale around Christmas, it has the same issue.

Let me know how I can help.

**Comment by SoundSailor** [comment]
Just tried turning off HDMI-CEC and now the screensaver is working.

**Comment by techadvisor1997** [comment]
i was having the problem with one roku tv but not another.  the common denominator was the wifi signal was lower for the tv that would display no images.  works fine on tv closer to the ap.
________________________________
